governance-proposal/TSC-Policy-Draft.md

```diff
@@ -0,0 +1,31 @@
+## ***DRAFT for Public Comment***
+
+# Node.js Foundation TSC Policy
+
+The Board of Directors of Node.js Foundation has set in place the following policies to govern the operation of the Technical Steering Committee (TSC). These policies relate to release processes, technical scope, business goals, etc. The TSC will, to the best of its ability, adhere to these policies. In cases where the TSC makes a judgment that the goals of Node.js Foundation are better served by making exceptions to these policies, it is expected that the TSC will communicate these exceptions and indicate their reasons to the Board at the next Board meeting.
+
+The scope of projects chosen by the TSC will focus on the subsystems, modules, scripts, tools or infrastructure to support Node.js.
+
+Node.js Foundation will support projects focused on the following:
+* Code development
+* Platform Integration & Testing
+* Documentation
+* Collaboration with external projects
+* Other projects that the TSC determines will improve Node.js
+
+The following relate to the projects initiated by the TSC and the artifacts created therein.
+* **Singularity**: To the extent possible, there should be no overlap between the significant functions of the Core projects.  
+* **Cohesiveness**: The artifacts created within each Core project should connect appropriately to other Core Projects to form a cohesive system. It is understood that this will not apply to artifacts that are stand-alone by design or dependencies external to the Foundation.
+* **Non-interference**: The artifacts created should work in any configuration and not create negative interference with each other’s functionality.
+
+The following relate to the choice of projects, assignments of tasks and delivery of code.
+Simultaneous Release: The TSC is responsible for organizing a simultaneous release of appropriate projects at regular intervals.
+
+The following relate to the operation of the TSC.
+* **Communication**: All communication between and within the TSC and Projects will be in a fair, open and consistent fashion.
+* **Openness**: The TSC should ensure that all technical decisions are made in an open and transparent fashion.
+* **Responsive to Collaborators**: The TSC should ensure issues and needs of the Collaborator community are being addressed in a timely fashion. Collaborators may opt to elevate pull requests or issues to the TSC for discussion by assigning the `tsc-agenda` tag. This should be done where a pull request:
+ * has a significant impact on the codebase,
+ * is inherently controversial; or
+ * has failed to reach consensus amongst the Collaborators who are actively participating in the discussion.
+ * The TC should serve as the final arbiter where required.
```

Comment:
Good catch - it looks like a remnant of our copy/paste effort from the io.js documents.